hack/boilerplate/boilerplate.py

```diff
@@ -215,7 +215,7 @@ def get_regexs():
     # strip #!.* from scripts
     regexs["shebang"] = re.compile(r"^(#!.*\n)\n*", re.MULTILINE)
     # Search for generated files
-    regexs["generated"] = re.compile('DO NOT EDIT|Do not edit it manually!')
+    regexs["generated"] = re.compile('DO NOT EDIT')
```

Comment:
is that rollback intended?

Reply:
I assume we're not using the python script anymore?

Reply:
the upstream openapi-gen adds a `DO NOT EDIT` so the `|Do not edit it manually!` (which I added 1 or two commits ago) is reverted.

Reply:
Fun, so that came from us somehow? :)

Reply:
came from our custom main.go